src/functions-and-arrays.js

```diff
@@ -29,9 +73,43 @@ const wordsUnique = [
   'bring'
 ];
 
+function uniquifyArray(arr) {
+  let newArr = [];
+  for (let i=0, word=arr.length; i<word ; i++)
+  if(newArr.indexOf(arr[i]) === -1 && arr[i] !== '')
+  newArr.push(arr[i]);
+  return newArr;
+}
+uniquifyArray(words)
+
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
+// Función que definimos pero que cuando la pasamos por Repl.it no funciona. ¿Nos podrías indicar que estaría fallando?
+function doesWordExist(arr, word){
+  words.forEach(function(x) {
+    if(words[x] === word) {
+      return "si"
+    } else {
+      return false;
+    }
+  })
+  }
```

Comment:
Aquí también hay un mix de "arr" y "words" 😉 
Lo ideal sería que devolvieses true dentro del for loop, cuando la palabra actualmente iterada (words[x]) es igual a la palabra buscada (word);
Si no devuelve true, es que la palabra no estaba en el array, y entonces devolvemos false al salir del for loop:

function doesWordExist(arr, word) {
  for (let i=0 ; i<arr.length; i++) {
    if (arr[i] === word) {
      return true;
    } 
  }
  return fa